libbeat/outputs/elasticsearch/config_otel.go

```diff
@@ -79,6 +80,23 @@ func ToOTelConfig(output *config.C) (map[string]any, error) {
 		return nil, fmt.Errorf("these configuration parameters are not supported %+v", temp)
 	}
 
+	// apply preset here
+	// It is important to apply preset before unpacking the config, as preset can override output fields
+	preset, err := output.String("preset", -1)
+	if err == nil {
+		// Performance preset is present, apply it and log any fields that
+		// were overridden
+		overriddenFields, presetConfig, err := applyPreset(preset, output)
+		if err != nil {
+			return nil, err
+		}
+		logp.Info("Applying performance preset '%v': %v",
+			preset, config.DebugString(presetConfig, false))
+		for _, field := range overriddenFields {
+			logp.Warn("Performance preset '%v' overrides user setting for field '%v'", preset, field)
+		}
```

Comment:
[Suggestion]
Use a single log entry with a list of fields:
```suggestion
		logp.Warn("Performance preset '%v' overrides user setting for field(s): %s",
			preset, strings.Join(overriddenFields, ","))
```